packages/api-tests/test/window.core.spec.js

```diff
@@ -694,6 +694,69 @@ describe('WindowCore API', function(done) {
         return chainPromises(steps);
       });
     }
+
+    it.only('Should emit a message event in the window #ssf.Window.postMessage #ssf.Window.postMessage', function() {
```

Comment:
Also one of those tags should be `#ssf.WindowCore.postMessage` so it can link it to the correct class in the docs